lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -65,6 +65,31 @@ def custom_build_settings
             settings
           end
 
+          # Filters the given resource file references discarding empty paths which are
```

Comment:
Hm. I don't have any specific suggestions. I would err on the side of verbosity, if you feel like a longer explanation might work better :)